src/lib/Microsoft.Health.Fhir.Ingest/Microsoft.Health.Fhir.Ingest.csproj

```diff
@@ -6,6 +6,7 @@
     <AssemblyName>Microsoft.Health.Fhir.Ingest</AssemblyName>
     <RootNamespace>Microsoft.Health.Fhir.Ingest</RootNamespace>
     <langversion>7.3</langversion>
+    <Configurations>Release;Debug</Configurations>

```

Comment:
Curious why this was added.  Can this be reverted?

Reply:
Hmmm must have been some Visual Studio weirdness. I'll revert it